dashboards/gateway-api-inference-extension/inference-extension-prometheus.json

```diff
@@ -0,0 +1,763 @@
+{
+    "displayName": "Inference Extension Prometheus Overview",
+    "dashboardFilters": [],
```

Comment:
For consistency with vLLM, Triton, etc, can you add cluster / location / namespace as filters for convenient drilling down?